### PR TITLE
controller/v14: update imports and packages

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/giantswarm/issues/3593

### CODE REVIEW

File: service/controller/v14/version_bundle.go [hunk 1]
```diff
@@ -1,64 +1,14 @@
...
-			},
-			{
-				Name:    "kubernetes",
-				Version: "1.10.4",
-			},
-			{
-				Name:    "nginx-ingress-controller",
-				Version: "0.12.0",
-			},
-		},
```
Comment (xh3b4sd) [inline service/controller/v14/version_bundle.go]: Components can stay. Changelogs should have one dummy item so you can fill it right away.

**Review by xh3b4sd — CHANGES_REQUESTED** [review]
https://github.com/giantswarm/kvm-operator/blob/f05032c6518c47c4dc7c93ff2f4aca2e0210e3d9/service/version_bundles.go is missing.

**Comment by JosephSalisbury** [comment]
@xh3b4sd PTALA

**Review by xh3b4sd — CHANGES_REQUESTED** [review]
https://github.com/giantswarm/kvm-operator/blob/f05032c6518c47c4dc7c93ff2f4aca2e0210e3d9/service/version_bundles.go is missing.

**Review by xh3b4sd — APPROVED** [review]

**Comment by JosephSalisbury** [comment]
https://media.giphy.com/media/26BRwtmWTmy2xsbmg/giphy.gif

**Comment by xh3b4sd** [comment]
like_a_boss_army

**Pull request merged**